sum-reward/src/mappings/Reward.ts

```diff
@@ -0,0 +1,39 @@
+import {SumReward} from '../types/models/SumReward';
+import {SubstrateEvent} from "@subql/types";
+import {Balance} from '@polkadot/types/interfaces';
+
+export async function handleBond(event: SubstrateEvent): Promise<void> {
+    const {event: {data: [account, balance]}} = event;
+    let entity:SumReward;
+    try{
+        entity = await SumReward.get(account.toString());
+    }catch (e) {
+        entity = new SumReward(account.toString());
+        entity.accountReward = BigInt(0);
+        entity.accountSlash = BigInt(0);
+        entity.accountTotal = BigInt(0);
+        await entity.save();
+    }
+}
+
+
+export async function handleReward(event: SubstrateEvent): Promise<void> {
+    const {event: {data: [account, balance]}} = event;
+    const entity = await SumReward.get(account.toString());
+    const newReward = balance as Balance;
```

Comment:
it is only used once, so I think you can use it inline directly. same for handleSlash